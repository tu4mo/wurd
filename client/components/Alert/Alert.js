import React from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components'

const StyledAlert = styled.div`
  background-color: #f2dede;
  border-radius: var(--radius-sm);
  color: #a94442;
  display: flex;
  justify-content: center;
  padding: var(--spacing-sm);
`

const Alert = ({ message }) => <StyledAlert>{message}</StyledAlert>

Alert.propTypes = {
  message: PropTypes.string
}

export default Alert
