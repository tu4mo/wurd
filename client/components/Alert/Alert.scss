.alert {
  background-color: #f2dede;
  border-radius: var(--radius-sm);
  color: #a94442;
  display: flex;
  justify-content: center;
  padding: var(--spacing-sm);
}
