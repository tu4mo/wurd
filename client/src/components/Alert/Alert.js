import React from 'react'
import PropTypes from 'prop-types'
import './Alert.scss'

const Alert = ({ message }) => {
  return (
    <div className="alert">
      {message}
    </div>
  )
}

Alert.propTypes = {
  message: PropTypes.string
}

export default Alert
