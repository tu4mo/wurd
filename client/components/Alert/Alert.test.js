import React from 'react'
import Alert from './Alert'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

it('renders correctly', () => {
  const tree = renderer.create(<Alert message="Hello World!" />).toJSON()
  expect(tree).toMatchSnapshot()
})
