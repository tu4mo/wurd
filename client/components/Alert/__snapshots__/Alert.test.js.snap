// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  background-color: #f2dede;
  border-radius: var(--radius-sm);
  color: #a94442;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: var(--spacing-sm);
}

<div
  className="c0"
>
  Hello World!
</div>
`;
